### PR TITLE
docs: modify layout style

### PR DESCRIPTION
**Pull request by chenxch**
- @media(min-width: 1440px)
 before:
image

 after:
image

- @media(min-width: 1680px)  in 1920
 before:
 
image

after:
image

Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Comment by kooriookami** [comment]
image

These two gap is not equal. The left gap is more big than the right.

**Review by kooriookami — APPROVED** [review]

**Review by kooriookami — APPROVED** [review]

**Review by kooriookami — APPROVED** [review]

File: docs/.vitepress/vitepress/styles/content/table-of-content.scss [hunk 1]
```diff
@@ -43,7 +61,9 @@
       .toc-item {
         margin-top: 10px;
         font-size: 11px;
-        color: var(--text-color-lighter);
+        text-overflow: ellipsis;
+        overflow: hidden;
+        white-space: nowrap;
```
[the remark this answers is not in the document]
Comment (kooriookami) [reply docs/.vitepress/vitepress/styles/content/table-of-content.scss]: Yes, it's a merge mistake.

File: docs/examples/alert/description.vue [hunk 2]
```diff
@@ -1,7 +1,10 @@
...
-  />
+  <div style="max-width: 600px">
+    <el-alert
+      title="with description"
+      type="success"
+      description="This is a description."
+    />
+  </div>
 </template>
+<script setup lang="ts"></script>
```
Comment (btea) [inline docs/examples/alert/description.vue:10]: Is this script tag without content necessary?

Comment (kooriookami) [reply docs/examples/alert/description.vue:10]: Oh, it's added automatically when saving the file.

Comment (btea) [reply docs/examples/alert/description.vue:10]: Is this automatically added by the plugin you installed? Perhaps we can remove it?

Comment (kooriookami) [reply docs/examples/alert/description.vue:10]: Let me try.

**Review by kooriookami — APPROVED** [review]

**Review by btea — APPROVED** [review]

**Review by kooriookami — APPROVED** [review]

**Pull request merged**